Automattic/woocommerce-payments
# Use `type_is_in` filter from the client to filter by multiple transaction types.

Fixes #8783 

#### Changes proposed in this Pull Request


#### Testing instructions

- Test with server PR https://github.com/Automattic/transact-platform-server/pull/6944
- Enable Payment activity widget. See that the `View Report` links are 
     - loading with correct URL e.g `https://....&path=%2Fpayments%2Ftransactions&filter=advanced&...type_is_in%5B0%5D=dispute&type_is_in%5B1%5D=dispute_reversal` , 
     - does not show `type` in search field 
     - but displays the correct transaction types

![image](https://github.com/user-attachments/assets/540aca4c-2eb6-44cf-af21-bac2d52faac8)


-------------------


**Post merge**
